.gitignore

```diff
@@ -1 +1,5 @@
 deploy/public
+/node_modules
+package-lock.json
```

Comment:
Should we definitely be ignoring the package lock file?

Reply:
I would add it.